mmdata.c

```diff
@@ -1170,6 +1203,34 @@ nmbrString *nmbrSpace(long n)
   return (sout);
 }
 
+/* Search for the nth occurrence of string2 in string1 */
+long nmbrInstrN(long start_position, long occ, nmbrString *string1,
+  nmbrString *string2, long start2, long length2)
+{
+  if (start_position < 1) start_position = 1;

```

Comment:
This means: If garbage is provided in parameter start_position, then I sanitize it to something more useful, hoping for the best.  This kind of fault tolerance supports a sloppy programming on the caller's side.  Better throw an exception, or call bug()